content/applications/general/calendars/outlook/outlook_calendar.rst

```diff
@@ -1,103 +1,149 @@
+on how to set up an Azure AD Tenant (also called an *environment*), which is a representation of an
+organization to manage and register apps.
+
+Then, `Register an Application
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_,
+choosing the appropriate supported account type. Users who wish to connect their Outlook calendar
+to Odoo should select the :guilabel:`Accounts in any organizational directory (Any Azure AD
+directory - Multitenant and personal Microsoft accounts (e.g. Skype, Xbox)` option for
+:guilabel:`Supported account types`.
 
-   .. image:: outlook_calendar/platform_configurations.png
-      :align: center
-      :alt: Adding a platform in Microsoft Azure for Outlook calendar sync in Odoo
+
+When configuring the platform settings, choose :guilabel:`Web`.
+
+.. image:: outlook_calendar/platform_configurations.png
+   :align: center
+   :alt: Adding a platform in Microsoft Azure for Outlook calendar sync in Odoo.
 
 .. important::
-   | For the **Redirect URI restrictions**, copy your Odoo database URI followed by
-     ``/microsoft_account/authentication``. Example:
+   For the :guilabel:`Redirect URI restrictions`, copy the Odoo database URI (URL) followed by
+   `/microsoft_account/authentication`. For example, 
+   `(https://www.companyname.odoo.com/microsoft_account/authentication)`.
+
+   .. image:: outlook_calendar/redirect_URIs.png
+      :align: center
+      :alt: Odoo's database URI that is accepted when Microsoft returns authentication.
 
-      .. image:: outlook_calendar/redirect_URIs.png
-         :align: center
-         :alt: Odoo's database URI that is accepted when microsoft returns authentication
+   For more information on the restrictions and limitations of URIs, `check this page <https://docs.microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
 
-   | For more information on the restrictions and limitations of URIs, `check this page <https://docs.microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
+Regarding credentials, the user **must** :guilabel:`Add a client secret`, which allows Odoo to
+authenticate itself, requiring no interaction from the user's side. :guilabel:`Certificates` are
+optional.
 
-.. note::
-   Regarding credentials, you *must* **Add a client secret**, called *Client Secret* in Odoo, which
-   allows Odoo to authenticate itself, requiring no interaction from your side. *Certificates* are
-   optional.
+Since resetting the synchronization can be tricky, Odoo recommends setting the maximum allowed
+:guilabel:`expiration date` for the Client Secret (24 months), so there is no need to
+re-synchronize soon. 
 
 Configuration in Odoo
 =====================
 
-Go to general *Settings* and activate the *Outlook Calendar* feature.
+In the Odoo database, go to :menuselection:`Settings --> General Settings` and activate the
+:guilabel:`Outlook Calendar` feature.
 
 .. image:: outlook_calendar/outlook_calendar.png
```

Comment:
Media filenames should be separated with [hypens](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files), not underscores.

As well, does this need to be 1200px?